src/Application.vala

```diff
@@ -166,6 +173,36 @@ namespace Maya {
             Calendar.EventStore.get_default ().delete_trashed_calendars ();
         }
 
+        public async bool ask_for_background () {
+            string handle = active_window != null ? yield ((MainWindow) active_window).export () : "";
+            string[] cmd = { "io.elementary.calendar", "--background" };
+            bool can_background = true;
+
+            var options = new HashTable<string, Variant> (str_hash, str_equal);
+            options["handle_token"] = Portal.generate_token ();
+            options["reason"] = _("Calendar wants to run in the background and initialize with the session for events notifications");
```

Comment:
More in line with the copy we just merged to Mail

```suggestion
            options["reason"] = _("Calendar will automatically start when this device turns on and run when its window is closed so that it can send event notifications.");
```